XML/TestCases/StressTests.xml

```diff
@@ -124,4 +124,33 @@
 		<Tags>stress,network,sriov</Tags>
 		<Priority>2</Priority>
 	</test>
+	<test>
+		<testName>STRESS-STORAGE-4K-IO-WITHVERIFY</testName>
+		<testScript>PERF-STORAGE-MULTIDISK-RAID0-FIO.ps1</testScript>
+		<setupType>OneVM12Disk</setupType>
+		<AdditionalHWConfig>
+			<DiskType>Managed</DiskType>
+			<StorageAccountType>PERF_STORAGE_ACCOUNT_TYPE</StorageAccountType>
+		</AdditionalHWConfig>
+		<setupScript>.\Testscripts\Windows\SETUP-Check-PowerPlan.ps1,.\Testscripts\Windows\AddHardDisk.ps1</setupScript>
+		<files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\perf_fio.sh,.\Testscripts\Linux\fio_jason_parser.sh,.\Testscripts\Linux\JSON.awk,.\Tools\gawk</files>
+		<TestParameters>
```

Comment:
It is nice to have ioruntime parameter for future extension, like "<param>ioruntime=FIO_STRESS_RUNTIME</param>"